src/app/clusters/operational-credentials-server/dac_chain.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+// TODO: Implement a mechanism to handle and secure DAC/PAI/PAA certificates and corresponding keypairs
+// Currently this is hard-coded to demonstrate Attestation Flow
+
+const uint8_t pai_certificate[] = {
```

Comment:
Could you please add the PEM of test PAA cert and associated private key used,as comments?

Reply:
Updated